.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Issue Number:
+- Here goes the issue number with a #
+-----------------------------------------------------------
+## Issue Description:
+- Describe the issue fixed here
+-----------------------------------------------------------
+## Checklist:

```

Comment:
In the `checklist` section, 
+ Tested the endpoint on sample data.
+ No conflicts
+ Have mentioned the endpoint in the `ENDPOINTS.md` file.